app/controllers/spree/admin/payments_controller.rb

```diff
@@ -35,15 +35,16 @@ def create
             return
           end
 
-          authorize_stripe_sca_payment
-
           if @order.completed?
-            @payment.process!
+            authorize_stripe_sca_payment
```

Comment:
I did a deep dive into this method yesterday and I fail to understand why we skip non-StripeSCA payments like `@payment.payment_method.class == Spree::Gateway::StripeSCA` Why is it that we don't want to authorize PayPal payments, for instance? does anyone know? @luisramos0 ?

Reply:
I'm going to mark this resolved; @sauloperez has started to refactor and address this in https://github.com/openfoodfoundation/openfoodnetwork/pull/7809